docs/notebook_tutorial/ocr.ipynb

```diff
@@ -148,8 +156,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 27,
-   "metadata": {},
+   "execution_count": 12,
+   "metadata": {
+    "tags": [
+     "ocr_token_processor"
```

Comment:
There is no description of the OcrTokenProcessor here, it is a good place to discuss the links here.

Reply:
`tag` is a feature for unit test to locate code cell. I added class docstring for `OcrTokenProcessor`

Reply:
yeah I simply comment around this area. What I mean is that the `ocr_token_processor` is not described. Ideally, a markdown cell is better than a inline docstring (visually more clearly)

Reply:
https://hyp.is/oJgooB_bEe2nFrPRFh84UQ/asyml-forte.readthedocs.io/en/latest/notebook_tutorial/ocr.html

Reply:
btw, I think we can try out the on-page comment here: 

https://asyml-forte--889.org.readthedocs.build/en/889/notebook_tutorial/ocr.html#Pipeline-Components-for-OCR

could you see them?

Reply:
Yes. It requires two steps. login and change the group to `Petuum`.

Reply:
thanks. maybe helpful for us to communicate on this.